### PR TITLE
Small fixes + inkCharacterEvent

### PR DESCRIPTION
**Pull request by psiberx**
- Various small fixes
- Decoded `inkCharacterEvent`
- Added constexpr `TweakDBID` (static and header-only)
- Added `TagList` methods

### CODE REVIEW

**Review by WopsS — DISMISSED** [review]
Thanks :D

Should I keep this open or can I merge it?

Edit: Nvm, I see now it is a draft.

**Comment by psiberx** [comment]
It's ready .)

File: include/RED4ext/NativeTypes.hpp [hunk 1]
```diff
@@ -56,9 +57,31 @@ struct TweakDBID
     };
 #pragma pack(pop)
 
-    TweakDBID() noexcept = default;
-    TweakDBID(uint64_t aValue) noexcept;
-    TweakDBID(uint32_t aNameHash, uint8_t aNameLength) noexcept;
+    constexpr TweakDBID() noexcept = default;
```
Comment (WopsS) [inline include/RED4ext/NativeTypes.hpp:60]: Do we care if the value is not inited to 0?

File: include/RED4ext/Scripting/Natives/redTagList.hpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+{
+    static constexpr const char* NAME = "redTagList";
+    static constexpr const char* ALIAS = NAME;
+
+    operator bool() const noexcept;
+
+    [[nodiscard]] bool IsEmpty() const;
+    [[nodiscard]] bool HasTag(CName aTag) const;
+    [[nodiscard]] bool HasTags(const DynArray<CName>& aTags) const;
+    [[nodiscard]] bool HasTags(const TagList& aTags) const;
```
Comment (WopsS) [inline include/RED4ext/Scripting/Natives/redTagList.hpp:21]: What about
```suggestion
    [[nodiscard]] bool Contains(CName aTag) const;
    [[nodiscard]] bool Contains(const DynArray<CName>& aTags) const;
    [[nodiscard]] bool Contains(const TagList& aTags) const;
```
?
Same for `Add` and `Remove`.

**Review by WopsS — CHANGES_REQUESTED** [review]
Nice! Only a few questions.

File: include/RED4ext/NativeTypes.hpp (hunk 1 above)
Comment (psiberx) [reply include/RED4ext/NativeTypes.hpp]: Yes :)

**Comment by WopsS** [comment]
Thanks again! ❤️

**Review by WopsS — APPROVED** [review]

**Pull request merged**